### PR TITLE
[Marvell] Add support to new platforms

### PR DESCRIPTION
**Pull request by pavannaregundi**
#### Why I did it
Add following variants of 98DX35xx and 98CX85xx(falcon) board to marvell build.

Board with external x86_64 CPU connected over PCI to 98DX35xx switching chip:
```
     Platform: x86_64-marvell_rd98DX35xx-r0
     HwSKU: rd98DX35xx
     ASIC: marvell
     Port Config: 32x1G + 16x2.5G + 6x25G
```
Board with external x86_64 CPU connected over PCI to 98CX8580 switching chip:
```
     Platform: x86_64-marvell_db98cx8580_32cd-r0
     HwSKU: db98cx8580_32cd
     ASIC: marvell
     Port Config: 32x400G
```
    
Board with external x86_64 CPU connected over PCI to 98CX8540 switching chip:
```
     Platform: x86_64-marvell_db98cx8540_16cd-r0
     HwSKU: db98cx8540_16cd
     ASIC: marvell
     Port Config: 16x400G
```
    
Board with external x86_64 CPU connected over PCI to 98CX8514 switching chip:
```
     Platform: x86_64-marvell_db98cx8514_10cc-r0
     HwSKU: db98cx8514_10cc
     ASIC: marvell
     Port Config: 48x25G + 8x100G
```

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added submodule changes in platform/marvell/sonic-platform-marvell and platform/marvell/mrvl-prestera. Add makefile changes for creating sonic-platform debs.
Added new HWSKU in device/marvell.

#### How to verify it
Build and verified ONIE and SONiC installs and checked to see port status UP.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by Blueve — APPROVED** [review]

File: device/marvell/x86_64-marvell_db98cx8580_32cd-r0/pmon_daemon_control.json
Comment (prgeor) [inline device/marvell/x86_64-marvell_db98cx8580_32cd-r0/pmon_daemon_control.json:1]: @pavannaregundi why are skipping all these PMON daemons?

**Review by prgeor — COMMENTED** [review]
@pavannaregundi why are we skipping all PMON daemons for all these platforms?

File: device/marvell/x86_64-marvell_db98cx8580_32cd-r0/pmon_daemon_control.json
Comment (pavannaregundi) [reply device/marvell/x86_64-marvell_db98cx8580_32cd-r0/pmon_daemon_control.json:1]: @prgeor Thanks for review. These are reference and development boards, hence these are skipped.

**Comment by pavannaregundi** [comment]
> @pavannaregundi why are we skipping all PMON daemons for all these platforms?

@prgeor Thanks for review. These are reference and development boards, hence these are skipped.

**Pull request merged**